Mmansurova/abc 414

Mil-m opened the pull request.

Raheelaaslam1024 commented:
1.) CSS is changed for links and tracking system filter as well.

![Screenshot 2025-03-12 at 10 39 53](https://github.com/user-attachments/assets/d7509f8c-b211-454a-9789-faaeaeabcca1)

Mil-m commented:
> Please address the following issues before we proceed with merging:
> 
> ## `http://localhost:4200/data_portal/1594226`
> * Add a horizontal line (`<hr>`) above the 'Tracking System' filters in the Symbionts tab.
> * Clicking on the BioSampleID in the symbionts tab displays a page with 3 field names but no data
> * In DToL, related tables (e.g., Relationships) have sorting enabled, but ERGA does not. This is not mandatory at the moment
> * In DToL, the number of filters is limited to three, with a collapse/expand option to show additional filter entries.
> 
> ## `http://localhost:4200/data_portal/753153`
> * In the sampling map, if we click on number 4, the icon is not properly displayed
> * In DToL, 'Rutpela maculata' has an NBN Atlas Reference, but this is missing for ERGA
> 
> ## `http://localhost:4200/data_portal/2714341`
> * When selecting 'Unknown' for Sex or Organism Part filters, an empty result set is displayed with the message: _No data matching the filter "".  If the value is 'Unknown', the filters should not be displayed.
> * The GeoLocation map appears as an empty grey square. If no data is available, the tab should not be displayed.

as far as I know we don't need the 'NBN Atlas Reference' more

Mil-m commented:
> 1.) CSS is changed for links and tracking system filter as well.
> 
> ![Screenshot 2025-03-12 at 10 39 53](https://private-user-images.githubusercontent.com/160017141/421833708-d7509f8c-b211-454a-9789-faaeaeabcca1.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NDE3OTE3MjEsIm5iZiI6MTc0MTc5MTQyMSwicGF0aCI6Ii8xNjAwMTcxNDEvNDIxODMzNzA4LWQ3NTA5ZjhjLWIyMTEtNDU0YS05Nzg5LWZhYWVhZWFiY2NhMS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUwMzEyJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MDMxMlQxNDU3MDFaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT00NzVkZmRhMmIyZGNkNWM0M2JkMmRmOWE0ZTQyMGI1YjI1Nzk1NThiOWMwMmY0ODJmYzkzM2RmMWJhOTgyNjE0JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.piPto42j9_uhXt3KRHJEUadsCznUBfr3z_k0VLfQQ2Q)

this was fixed by adding MatButtonModule

Mil-m commented:
> Please address the following issues before we proceed with merging:
> 
> ## `http://localhost:4200/data_portal/1594226`
> * Add a horizontal line (`<hr>`) above the 'Tracking System' filters in the Symbionts tab.
> * Clicking on the BioSampleID in the symbionts tab displays a page with 3 field names but no data
> * In DToL, related tables (e.g., Relationships) have sorting enabled, but ERGA does not. This is not mandatory at the moment
> * In DToL, the number of filters is limited to three, with a collapse/expand option to show additional filter entries.
> 
> ## `http://localhost:4200/data_portal/753153`
> * In the sampling map, if we click on number 4, the icon is not properly displayed
> * In DToL, 'Rutpela maculata' has an NBN Atlas Reference, but this is missing for ERGA
> 
> ## `http://localhost:4200/data_portal/2714341`
> * When selecting 'Unknown' for Sex or Organism Part filters, an empty result set is displayed with the message: _No data matching the filter "".  If the value is 'Unknown', the filters should not be displayed.
> * The GeoLocation map appears as an empty grey square. If no data is available, the tab should not be displayed.

http://localhost:4200/data_portal/1594226

<img width="292" alt="Screenshot 2025-03-12 at 15 01 00" src="https://github.com/user-attachments/assets/7860a0a1-673c-4a5d-bfde-ecb17383de40" />

filters part looks in this way for now

* Clicking on the BioSampleID in the symbionts tab displays a page with 3 field names but no data - fixed (we will see only the sample name as it made in the DToL)

for example
http://localhost:4200/organism/SAMEA110701669

* In DToL, the number of filters is limited to three, with a collapse/expand option to show additional filter entries. - fixed

http://localhost:4200/data_portal/753153

* In the sampling map, if we click on number 4, the icon is not properly displayed - the icon will be shown after the cleaning cache in browser

* In DToL, 'Rutpela maculata' has an NBN Atlas Reference, but this is missing for ERGA - added

http://localhost:4200/data_portal/2714341

* When selecting 'Unknown' for Sex or Organism Part filters, an empty result set is displayed with the message: _No data matching the filter "".  If the value is 'Unknown', the filters should not be displayed. - these filters work as in the Data Portal tab

* The GeoLocation map appears as an empty grey square. If no data is available, the tab should not be displayed. - fixed

yroochun commented:
Code is ready to be merged
